chained/tls_file_cache.go

```diff
@@ -62,26 +64,39 @@ func PersistSessionStates(configDir string) {
 	})
 }
 
+func isSameLibraryVersion(rows []string) bool {
+	if len(rows) > 0 && strings.HasPrefix(rows[0], libraryVersionString) {
+		fileLibraryVersion := strings.TrimPrefix(rows[0], libraryVersionString)
+		log.Debugf("tls_session_states LibraryVersion %v", fileLibraryVersion)
+		log.Debugf("current LibraryVersion %v", common.LibraryVersion)
+		if fileLibraryVersion == common.LibraryVersion {
+			return true
+		}
+	} else {
+		log.Debugf("%v string not found in tls_session_states", libraryVersionString)
+	}
+	return false
+}
```

Comment:
We only need to check the first line, so we don't need to pass in all rows. Since `rows` is a slice, you can call `isSameLibraryVersion(rows[0])` and it will be set to the empty string. 

We also should check if `fileLibraryVersion >= common.LibraryVersion`, in case a user skips this version and upgrades to the next one from an older version.

Reply:
This part isn't absolutely necessary, but best practices for go is to return early. So this can be simplified to:
```
func isSameLibraryVersion(line string) bool {
	if !strings.HasPrefix(line, libraryVersionString) {
		return false
	}
	fileLibraryVersion := strings.TrimPrefix(line, libraryVersionString)
	log.Debugf("tls_session_states LibraryVersion %v", fileLibraryVersion)
	return fileLibraryVersion >= common.LibraryVersion 
}
```